pkg/queue/cluster_queue_impl.go

```diff
@@ -221,6 +222,22 @@ func (c *clusterQueueBase) DumpInadmissible() (sets.Set[string], bool) {
 	return elements, true
 }
 
+func (c *clusterQueueBase) Snapshot() ([]*workload.Info, bool) {
```

Comment:
This function does not need to return the bool, just the list of workloads

Reply:
fixed https://github.com/kubernetes-sigs/kueue/pull/1069/commits/59446b99221c239140ce79325634fcb5ec9cee27